ksql-functional-tests/src/test/resources/query-validation-tests/explode.json

```diff
@@ -0,0 +1,47 @@
+{
+  "comments": [
+    "Tests for the EXPLODE table function"
+  ],
+  "tests": [
+    {
+      "name": "explode array with values",
+      "statements": [
+        "CREATE STREAM TEST (MY_ARR ARRAY<BIGINT>) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT EXPLODE(MY_ARR) VAL FROM TEST;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 0, "value": {"ID":  0, "MY_ARR": [1, 2]}},
+        {"topic": "test_topic", "key": 1, "value": {"ID":  1, "MY_ARR": [3, 4]}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": "0", "value": {"VAL": 1}},
+        {"topic": "OUTPUT", "key": "0", "value": {"VAL": 2}},
+        {"topic": "OUTPUT", "key": "1", "value": {"VAL": 3}},
+        {"topic": "OUTPUT", "key": "1", "value": {"VAL": 4}}
+      ]
+    },
+    {
+      "name": "explode empty array",
```

Comment:
Personally, I'd combine this with the test case above, i.e. have an input row with an empty array.  Also, add an input row with a null array.

Reply:
I think it's better not to do much in each test, that way when one fails its simpler to see what its actually failing on.

Reply:
As a general principle, I'm in total agreement.  One unit test should test one thing.

However, these are not unit tests, they a functional tests and they're a little more expensive to setup and run.  They also get run against multiple versions of KSQL.  Hence, in this case, I'm suggesting we combine the two tests to keep our test run time in check.

Reply:
changed